latest/SSC/CGI/keywords/cgi_optical_config.yaml

```
@@ -0,0 +1,176 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/SSC/CGI/keywords/cgi_optical_config-1.0.0
+
+title: Optical configuration
+extName: SSC
+
+type: object
+properties:
+  SPAM_H:
+    title: SPAM +H position
+    description: SPAM +H position (um)
+    type: number
+    archive_catalog:
+      datatype: float
+      destination: [CGIExposure.spam_h, CGIMosiac.spam_h]
+  SPAM_V:
+    title: SPAM +V Position
+    description: SPAM +V Position (um)
+    type: number
+    archive_catalog:
+      datatype: float
+      destination: [CGIExposure.spam_v, CGIMosaic.spam_v]
+  SPAMNAME:
+    title: Closest SPAM named position
+    description: Closest SPAM named position
+    type: string
+    maxLength: 128
+    enum:
+      - TODO
```

Right now this indicates the `SPAMNAME` can only be `TODO` Lets comment the enum for now until we get clarification about what if any values should be enumerated
```suggestion
    # Waiting on specific enum values
    # enum: []
```